build.gradle

```diff
@@ -15,8 +15,10 @@ repositories {
 dependencies {
 	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
+	implementation 'org.springframework.boot:spring-boot-starter-thymeleaf'
 	runtimeOnly 'org.postgresql:postgresql'
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    developmentOnly "org.springframework.boot:spring-boot-devtools"
```

Comment:
Use single quotation and check indentation.

Reply:
Solved, I replaced the double quotes by single quotes and cleaned the white spaces by adding tabs